source/_integrations/huawei_solar.markdown

```diff
@@ -0,0 +1,128 @@
+---
+title: "Huawei Solar"
+description: "Instructions on how to connect your Huawei SUN2000 inverter to Home Assistant"
+ha_release: "2022.3"
+ha_category: 
+    - Sensor
+    - Energy
+ha_iot_class: "Local Polling"
+ha_config_flow: true
+ha_codeowners:
+  - '@wlcrs'
+ha_domain: huawei_solar
+---
+
+The Huawei Solar integration connects to the Modbus interface from a Huawei SUN2000(L) inverter and integrates it into Home Assistant.
+
+
+## Prerequisites
+
+You will need to either set a static IP on your Huawei Inverter or assign a static DHCP lease to it. Refer to 'Router connection settings' in 'Configuration' > 'Communication configuration' to set a static IP address. 

```

Comment:
You should specify if dongle is needed. According to the latest firmware update MODBUS won't be supported on inverter - https://forum.huawei.com/enterprise/en/forum.php?mod=redirect&goto=findpost&ptid=789585&pid=4632813. Also take a look at https://skyboo.net/2022/02/huawei-sun2000-why-using-a-usb-dongle-for-monitoring-is-not-a-good-idea/

If understand correct your integration works only with inverter wifi or with dongle? This should be specified.
What firmware version is needed on both devices?
BTW can we use MODBUS to ETH converter without using wifi? https://www.waveshare.com/rs485-to-eth.htm or attach LAN cable directly do dongle?
FusionSolar app should work with this integration. If yes this could be mentioned.